apps/mobile/src/components/browser/browser-sheet/use-browser-search-state.ts

```diff
@@ -0,0 +1,60 @@
+import { useReducer } from 'react';
```

Comment:
Nice work on this 👍  It would be good to add some unit tests for the reducer states